src/components/ChatApp/MessageListItem/Feedback.react.js

```diff
@@ -101,7 +101,7 @@ class Feedback extends React.Component {
 				positiveFeedbackColor = UserPreferencesStore.getTheme()==='light' ? '#1685e5' : '#00ff7f';
 			}
 			else if(this.state.negative){
-				negativeFeedbackColor = UserPreferencesStore.getTheme()==='light' ? '#1685e5' : '#00ff7f';
+				negativeFeedbackColor = UserPreferencesStore.getTheme()==='light' ? '#ff0000' : '#00ff7f';
```

Comment:
That is too much red, better switch to `#d1462f`, not shouting at the user. 😄

Reply:
I will fix it but how will I provide deployment link

Reply:
Check http://surge.sh/ it is fast and easy, but remember before you deploy it, follow https://github.com/fossasia/chat.susi.ai#how-to-deploy

Reply:
Well we don't have an necessity of deployment, I said only if possible, else this is a little change which can be reviewed on basis of files changed, just make the changes.